sites/public/src/components/account/ConfirmationModal.tsx

```diff
@@ -37,7 +38,7 @@ const ConfirmationModal = (props: ConfirmationModalProps) => {
     try {
       await resendConfirmation(email)
 
-      setSiteAlertMessage(t(`authentication.createAccount.emailSent`), "success")
+      setToast(t(`authentication.createAccount.emailSent`), { variant: "success" })
```

Comment:
Hard to test this one since doesn't seem like its working (dev or in PR)
https://github.com/bloom-housing/bloom/assets/53269332/93a011fd-3776-45e9-b3b6-7027bf5fffa6